src/chart.zoom.js

```diff
@@ -41,32 +41,36 @@ function zoomIndexScale(scale, zoom, center) {
 
 }
 
-
 function zoomTimeScale(scale, zoom, center) {
 	var options = scale.options;
-	var minDelta, maxDelta;
-	var newDiff;
+	var range = scale.right - scale.left;
```

Comment:
Does it need to? http://www.chartjs.org/docs/#scales-time-scale says Time Scale can only be placed on the X axis.